apps/web/src/components/auth/LoginForm.tsx

```diff
@@ -59,6 +61,13 @@ export function LoginForm({ email, token }: Props) {
     try {
       const response = await mutateAsync(itemData);
       setToken((response as any).token);
+
+      const user = await getUser();
```

Comment:
I would suggest not doing LogRocket identify as part of a login form, since the user can be logged in from a lot of places. (OAuth, signup and etc...) 

You can move this to be part of the authentication hook where we set the user. Or similarly to intercom put it in the high level layout section such as `HeaderNav.tsx` (probably not the best place but at least will work for other sign in methods